apps/hash-api/src/storage/index.ts

```diff
@@ -92,36 +103,182 @@ export const setupStorageProviders = (
   return getUploadStorageProvider();
 };
 
+const isFileEntity = (entity: Entity): entity is Entity<FileProperties> =>
+  SYSTEM_TYPES.propertyType.fileStorageKey.metadata.recordId.baseUrl in
+    entity.properties &&
+  extractBaseUrl(fileUrlPropertyTypeUrl) in entity.properties;
+
+const isValidStorageType = (storageType: string): storageType is StorageType =>
```

Comment:
```suggestion
const isStorageType = (storageType: string): storageType is StorageType =>
```

Reply:
Updated in https://github.com/hashintel/hash/pull/3504/commits/3c6a56b91684afa4c40e0ef01a45de58210fc058